iocage_cli/update.py

```diff
@@ -29,11 +29,27 @@
 
 
 @click.command(
-    name="update",
-    help="Run freebsd-update to update a specified "
-    "jail to the latest patch level.")
-@click.argument("jail", required=True)
-def cli(jail):
+    name='update',
+    help='Run freebsd-update to update a specified '
+    'jail to the latest patch level.'
+)
+@click.option(
+    '--pkgs/--nopkgs', default=False,
+    help='Decide whether or not to update the pkg repositories and '
+         'all installed packages in jail.'
```

Comment:
We should make this true if the jail in question is a plugin jail, this flag should only be false for plugins if they specifically pass --nopkgs

Reply:
I am not sure but to be sure, is there potential for conflict between plugin updates and pkg updates in the same jail ?

Reply:
They're the same thing. Plugin updates currently are only pkg's + snapshot. So if they specifically disable that, all we should do in the plugin update process is snapshot and then update the world.